### PR TITLE
newstomato.com - Leftover

### PR DESCRIPTION
**Pull request by green1052**
image

i cant push master branch :(

### CODE REVIEW

**Comment by green1052** [comment]
test in https://www.newstomato.com/ReadNews.aspx?no=1132370&inflow=N

File: filters-share/specific_ELEMHIDE.txt [hunk 1]
```diff
@@ -29,6 +29,8 @@ gamemeca.com,m.ohmynews.com,m.khan.co.kr,m.dcinside.com##div[id^="ad"]
 ! ##div[id^="AD"]
 newsworker.co.kr##div[id^="AD"]
 !
+newstomato.com##.mn_banner
```
Comment (piquark6046) [inline filters-share/specific_ELEMHIDE.txt]: I cannot find the element on the website you provided.

**Review by piquark6046 — CHANGES_REQUESTED** [review]

File: filters-share/specific_ELEMHIDE.txt (hunk 1 above)
Comment (green1052) [reply filters-share/specific_ELEMHIDE.txt]: check mobile

Comment (green1052) [reply filters-share/specific_ELEMHIDE.txt]: i found kiwi browser

Comment (piquark6046) [reply filters-share/specific_ELEMHIDE.txt]: Found.

**Review by piquark6046 — APPROVED** [review]

**Pull request merged**